sources/packages/web/default.conf.template

```diff
@@ -1,5 +1,16 @@
 # nginx default configuration.
+
+log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                  '$status $body_bytes_sent "$http_referer" '
+                  '"$http_user_agent" "$http_x_forwarded_for"';
+
 server {
+  # Log to stdout
+  access_log /dev/stdout main;
+  
+  # Log to a file
+  access_log /var/log/nginx/access.log main;
```

Comment:
As we talked, let's avoid enabling the logs saved to a file right now and leave it for later to be enabled with some sort of file rotation.
Same for `sources/packages/web/nginx.conf`.
The idea is to keep the same behavior already in place with Kibana capturing the logs.